schema/sale_artworks.js

```diff
@@ -0,0 +1,22 @@
+import gravity from '../lib/loaders/gravity';
+import SaleArtwork from './sale_artwork';
+import {
+  GraphQLString,
+  GraphQLList,
+} from 'graphql';
+
+const SaleArtworks = {
+  type: new GraphQLList(SaleArtwork.type),
+  description: 'A list of Sale Artworks',
+  args: {
+    ids: {
+      type: new GraphQLList(GraphQLString),
+      description: 'A list of sale artwork ids',
+    },
```

Comment:
I think this is fine and probably easier to consume, but you also don't *really* have to expose an explicit schema for this if you can generate the appropriate query:

```
{
  my_first_sale_artwork: sale_artwork(id: "my_first_sale_artwork") { ... }
  my_second_sale_artwork: sale_artwork(id: "my_second_sale_artwork") { ... }
  # etc...
}
```